metricbeat/etc/fields.yml

```diff
@@ -344,3 +344,142 @@ redis:
               type: float
               description: >
                 User CPU consumed by the background processes
+system:
+  type: group
+  description: >
+    CPU stats for the host machine
```

Comment:
It has memory too. How about something like "System status metrics, like CPU and memory usage, collected from the operating system."